src/common/_modules/platform_service_bus/main.tf

```diff
@@ -0,0 +1,22 @@
+module "platform_service_bus_namespace" {
+  source  = "pagopa-dx/azure-service-bus-namespace/azurerm"
+  version = "~> 0.0"
+
+  environment = {
+    prefix          = var.prefix
+    env_short       = "p"
+    location        = var.location
+    app_name        = "platform"
+    instance_number = "01"
+  }
+
+  resource_group_name = var.resource_group_internal
+
+  subnet_pep_id                        = azurerm_subnet.platform_service_bus_namespace_subnet.id
```

Comment:
This should be the traditional subnet we set for the private endpoints `io-p-itn-pep-snet-01`. If I am right, it is defined in this configuration so you should already have the reference

Reply:
thank you, I missed it. replaced in https://github.com/pagopa/io-infra/pull/1520/commits/e145a80ebc29b877f695c1e3fcc002c503c8f1fa